pandas/core/frame.py

```diff
@@ -1598,8 +1597,9 @@ def to_feather(self, fname):
         from pandas.io.feather_format import to_feather
         to_feather(self, fname)
 
-    @Substitution(header='Write out column names. If a list of string is given, \
-it is assumed to be aliases for the column names')
+    @Substitution(header='Write out the column names. If a list of strings '
+                         'is given, it is assumed to be aliases for the '
+                         'column names')
```

Comment:
> As it is to_csv.__doc__ is all in one piece and I'm not inclined to disturb it.

Fair enough.  I can discuss with @jreback how we might want to proceed with this.

Regarding `Panel`, I do not know at this point.  My guess is `1.0`, but I would wait to see what @jreback or @jorisvandenbossche have to say about this.